behind-the-veil-siteroot/imports/ui/components/map/CenteredMap.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * File Description: A simple map component with a center at given location
+ * File version: 1.0
+ * Contributors: Laura
+ */
+
+import React, { useEffect, useState, useRef } from 'react';
+import mapboxgl from 'mapbox-gl';
+import 'mapbox-gl/dist/mapbox-gl.css'
+
```

Comment:
The file could possibly use some more comments, doesn't have to be too descriptive just representing what the implementation is doing